LICENSE

```diff
@@ -198,4 +198,4 @@
    distributed under the License is distributed on an "AS IS" BASIS,
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
    See the License for the specific language governing permissions and
-   limitations under the License.
\ No newline at end of file
+   limitations under the License.
```

Comment:
Weird. I can't tell what's different on this line. I suspect it's an extra newline, maybe added on save from your IDE as part of "auto-formatting", but it's weird that it doesn't render that way (I thought GitHub could render that better than this).

Reply:
Yep, auto formatting from my IDE. Seems consistent with other projects.